README.md

```diff
@@ -55,11 +55,19 @@ In a production MLOps solution, your ML pipelines need to be repeatable. So, we
 
 1. Clone the repository locally
 1. Install Python: `pyenv install`
-1. Install pipenv and pipenv dependencies: `make setup`
-1. Install pre-commit hooks: `cd pipelines && pipenv run pre-commit install`
+1. Install poetry and poetry dependencies: `make setup`
```

Comment:
This is redundant, no?

Reply:
No it's not, because in the Makefile we run pre-commit, but here we install it

Reply:
You are right!